Convert.pm

```diff
@@ -103,17 +101,20 @@ sub date2time {
 # Date format:  1999:260:03:30:01.542
 # 956245305.5 = eng_decom (unix) time at VCDU = 4324480 around 2000:111:15:41:46 (2000-04-20T15:42:50)
 # CXC Time from CCDM file is 72632569.96
-
     my $date = shift;
-    my $t1998 = @_ ? 0.0 : 956245305.5- 72632569.96 + 1.276; # 2nd argument implies Unix time not CXC time
-    my ($sec, $min, $hr, $doy, $yr) = reverse split ":", $date;
-
-    return ($doy*86400 + $hr*3600 + $min*60 + $sec) unless ($yr);
-
-    my ($mon, $day) = ydoy2md($yr, $doy);
-
-    $sec = 59 if ($sec > 59);
-    return timegm($sec,$min,$hr,$day,$mon-1,$yr) - $t1998;
+    # second argument can be used to request return time as unix seconds
+    my $want_unix_secs = shift;
+    # if this is delta date from the DOT, just cheat and use a full
+    # date relative to year 2000 (unix time would be moot here)
+    if ($date =~ /^\d{3}:\d{2}\:\d{2}:\d{2}\.\d{3}$/){
```

Comment:
@malgosias on the Perl side, in this still-open PR, I suggested just cheating, setting a year, and then finding a delta.

Reply:
I just changed this to use 2000:001 so it will at least be sensible.